packages/yarnpkg-pnpify/sources/generateSdk.ts

```diff
@@ -147,168 +262,91 @@ class Wrapper {
   }
 }
 
-export const generateEslintWrapper = async (pnpApi: PnpApi, target: PortablePath) => {
-  const wrapper = new Wrapper(`eslint` as PortablePath, {pnpApi, target});
-
-  await wrapper.writeManifest();
-
-  await wrapper.writeBinary(`bin/eslint.js` as PortablePath);
-  await wrapper.writeFile(`lib/api.js` as PortablePath);
-
-  await addVSCodeWorkspaceSettings(pnpApi, {
-    [`eslint.nodePath`]: npath.fromPortablePath(
-      ppath.dirname(ppath.dirname(ppath.dirname(
-        wrapper.getProjectPathTo(
-          `lib/api.js` as PortablePath,
-        ),
-      ))),
-    ),
-  });
-};
-
-export const generatePrettierWrapper = async (pnpApi: PnpApi, target: PortablePath) => {
-  const wrapper = new Wrapper(`prettier` as PortablePath, {pnpApi, target});
-
-  await wrapper.writeManifest();
-
-  await wrapper.writeBinary(`index.js` as PortablePath);
-
-  await addVSCodeWorkspaceSettings(pnpApi, {
-    [`prettier.prettierPath`]: npath.fromPortablePath(
-      wrapper.getProjectPathTo(
-        `index.js` as PortablePath,
-      ),
-    ),
-  });
-};
-
-export const generateTypescriptLanguageServerWrapper = async (pnpApi: PnpApi, target: PortablePath) => {
-  const wrapper = new Wrapper(`typescript-language-server` as PortablePath, {pnpApi, target});
+export const generateSdk = async (pnpApi: PnpApi, editors: typeof SUPPORTED_EDITORS | null, {report, base}: {report: Report, base: boolean}): Promise<void> => {
```

Comment:
I feel like the `| null` is somewhat inconvenient since we control the only callsite. Rather than pass `null`, why not just directly pass an empty set as parameter?

Reply:
You're right, it was a leftover from a few commits ago.